tests/test_controller.py

```diff
@@ -36,6 +55,7 @@ def vcr_cassette_dir(request):
     return str(Path(f"{__file__}").parent / "cassettes")
 
 
+@pytest.mark.skip(reason="need to update data with new vcr")
```

Comment:
If you are replacing these tests, why not delete them instead of skipping them?

Reply:
Removed